packages/zowe-explorer/src/job/ZosJobsProvider.ts

```diff
@@ -583,7 +609,7 @@ export class ZosJobsProvider extends ZoweTreeProvider implements IZoweTree<IZowe
         return;
     }
 
-    public async getUserJobsMenuChoice(): Promise<FilterItem> {
+    public async getUserJobsMenuChoice(): Promise<FilterItem | undefined> {
```

Comment:
Minor suggestion, might want to return `null` here instead as we are intentionally returning an absent value: https://262.ecma-international.org/13.0/#sec-null-value

But if `undefined` is the expected return type for this context here, then just ignore this comment

Reply:
I think undefined is for case where user presses escape and chooses nothing.

Reply:
Ah ok, in that case it wouldn't be an intentional absent value as it's something that the user performs. Sorry for the confusion